executor/slow_query.go

```diff
@@ -465,3 +509,151 @@ func ParseTime(s string) (time.Time, error) {
 	}
 	return t, err
 }
+
+type logFile struct {
+	file       *os.File  // The opened file handle
+	begin, end time.Time // The start/end time of the log file
+}
+
+// getAllFiles is used to get all slow-log need to parse, it is export for test.
```

Comment:
```suggestion
// getAllFiles is used to get all slow-log needed to parse, it is exported for test.
```

the function is exported? should it be started with `GetAllFiles`?

Reply:
the function doesn't export now.